src/Chainweb/Pact/Backend/Compaction.hs

```diff
@@ -70,7 +73,7 @@ import Data.LogMessage
 import Pact.Types.SQLite (SType(..), RType(..))
 import Pact.Types.SQLite qualified as Pact
 
-newtype TxId = TxId Int64
+newtype ITxId = ITxId Int64
```

Comment:
Whence?

Reply:
there is another type in scope that represents a TxId, and it's incompatible with this one. The 'I' means 'Int'. The reason this type exists is to provide some type-safety for TxIds used with Pact's SQLite API.